caluma/caluma_form/models.py

```diff
@@ -292,6 +295,24 @@ def copy(self, family=None, user=None):
 
         return new_document
 
+    @cached_property
+    def last_modified_answer(self):
+        return (
+            Answer.objects.filter(document__family=self).order_by("modified_at").last()
+        )
+
+    @property
+    def modified_content_at(self):
+        return self.last_modified_answer.modified_at
+
+    @property
+    def modified_content_by_user(self):
+        return self.last_modified_answer.modified_by_user
+
+    @property
+    def modified_content_by_group(self):
+        return self.last_modified_answer.modified_by_group
```

Comment:
`last_modified_answer` can be empty - for example in a freshly created document. These properties would cause an exception in those cases. Better to guard against it and return `None` if last_modified_answer is not set